File: enderecos/models/endereco.py
from django.db import models

class Enderecos (models.Model):
    cep=models.CharField(max_length=8,null=False)
    logradouro=models.CharField(max_length=50,null=False)
    numero=models.CharField(max_length=8)
    complemento=models.CharField(max_length=50,null=True)
    bairro=models.CharField(max_length=30)
    cidade=models.CharField(max_length=50)
    uf=models.CharField(max_length=2)
    
    def to_dict(self):
        return {
            'id': self.id,
            'cep': self.cep,
            'logradouro': self.logradouro,
            'numero': self.numero,
            'complemento': self.complemento,
            'bairro': self.bairro,
            'cidade': self.cidade,
            'uf': self.uf
        }

File: templates/static/js/meusJS/endereco.js
class Endereco{
    constructor(dados) {
        this.id=dados.id
        this.cep = dados.cep
        this.endereco = dados.logradouro
        this.numero=dados.numero
        this.complemento = dados.complemento
        this.bairro=dados.bairro
        this.cidade= dados.cidade
        this.uf=dados.uf
    }
    populaEndereco(identificacao){
        $('#cep'+identificacao).val(this.cep)
        $('#rua'+identificacao).val(this.endereco)
        $('#numero'+identificacao).val(this.numero)
        $('#complemento'+identificacao).val(this.complemento)
        $('#bairro'+identificacao).val(this.bairro)
        $('#cidade'+identificacao).val(this.cidade)
        $('#uf'+identificacao).val(this.uf)
    }
}
